lib/index.js

```diff
@@ -46,7 +47,7 @@ function VM (opts = {}) {
 
   if (this.opts.activatePrecompiles) {
     for (var i = 1; i <= 4; i++) {
-      this.trie.put(new Buffer('000000000000000000000000000000000000000' + i, 'hex'), new Account().serialize())
+      this.trie.put(Buffer.from('000000000000000000000000000000000000000' + i, 'hex'), new Account().serialize())
```

Comment:
Maybe `Buffer.from(new BN(i).toArray('be', 32))` is clear of intent? Speed doesn't matter much here.